src/js/classes/Table.js

```diff
@@ -202,10 +211,25 @@ export class RankedTable {
         const CellType = typeof(cell) == "number" ? NumberCell : TextCell;
         return new CellType(cell, this.classNames[j]);
       });
-      return new RankedBodyRow(cells, i + 1);
+      const isHidden = !this.matchesSearchTerm(row);
+      return new RankedBodyRow(cells, i + 1, isHidden);
     });
   }
 
+  matchesSearchTerm(row) {
+    if (this.searchTerms.length == 0) {
+      return true;
+    }
+    return this.searchTerms.some(searchTerm =>
+      row.some((value, i) =>
+        // Search term is selected from dropdown so
+        // is guaranteed to be equal to a value
+        this.searchCols[i] &&
+          value.toLocaleString().toLowerCase() === searchTerm.toLowerCase()
```

Comment:
Why `toLocaleString`? We'll only ever be using it for county names so that shouldn't be needed.

Reply:
I'm probably overthinking this but my thought process was - what if we're searching for a string where the data doesn't match the exact way someone would type it? toLocaleString most likely isn't the solution here so I'll remove it.

I want to be careful that we don't hard code assumptions like that we're only searching for county names (we do have a table where that's not quite the case - see comment below). If it's cheap lets try to have our code be flexible just in case the designs change.